kolibri/plugins/learn/assets/src/views/LibraryPage/NoResourcePage.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+
+  <LearnAppBarPage :appBarTitle="learnString('learnLabel')">
```

Comment:
Rather than duplicating the component completely - my suggestion was to make a component that represented the body of the ContentUnavailablePage component (everything _but_ the `LearnAppBarPage` component) and then reuse it in the LibraryPage and the ContentUnavailablePage.

Doing this has caused us to have two LearnAppBarPage components, one nested inside the other:

![image](https://github.com/user-attachments/assets/e841ae87-97cf-41cb-9c06-e62e23d85338)